### PR TITLE
feat: Comment List Feature

### PR DESCRIPTION
**Pull request by jimmyhogoboom**
### What does this PR do, or why is it needed?

Looks for a `comments` attribute on Content Items and includes a CommentListFeature when the value of that attribute is 'True'.

### What design trade-offs/decisions were made?

Rock administrators will need to modify their Content Channel Types to include a new boolean Item Attribute with a key of 'comments' (all lowercase).

### How do I test this PR?

The dev Rock environment already has the attribute added. You should see the CommentListFeature on each ContentItem in the following query:

```
query {
  contentChannels {
    childContentItemsConnection {
      edges {
        node {
          id
          title
          ... on WeekendContentItem {
            featureFeed {
              features {
                id
                ... on CommentListFeature {
                  id
                }
              }
            }
          }
        }
      }
    }
  }
}
```

You can then toggle the boolean to 'No' on any item to observe the CommentListFeature no longer being returned for that item.

### What automated tests did you write?

Added an assertion and to the snapshots for the existing features test.

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-data-connector-rock/src/content-items/data-source.js [hunk 1]
```diff
@@ -190,6 +177,13 @@ export default class ContentItem extends RockApolloDataSource {
       });
     }
 
+    const commentFeatures = get(attributeValues, 'comments.value', 'False');
+    if (commentFeatures === 'True') {
+      features.push(
+        Feature.createCommentListFeature({ nodeId: id, nodeType: __type })
```
Comment (vinnyjth) [inline packages/apollos-data-connector-rock/src/content-items/data-source.js:183]: awesome :D

**Review by vinnyjth — APPROVED** [review]
